packages/desktop-client/src/components/budget/index.tsx

```diff
@@ -106,31 +102,6 @@ function BudgetInner(props: BudgetInnerProps) {
     }
 
     run();
-
-    const unlistens = [
```

Comment:
No need for this since this is already handles by the global sync-event listener